Source/quests.cpp

```diff
@@ -752,6 +753,27 @@ void ResyncQuests()
 			}
 		}
 	}
+	if (Quests[Q_ZHAR].IsAvailable() && gbIsMultiplayer) {
+		Monster *zhar = FindUniqueMonster(UniqueMonsterType::Zhar);
+		if (zhar != nullptr && Quests[Q_ZHAR]._qvar1 != QS_ZHAR_INIT) {
+			zhar->flags |= MFLAG_QUEST_COMPLETE;
+
+			switch (Quests[Q_ZHAR]._qvar1) {
+			case QS_ZHAR_ITEM_SPAWNED:
+				zhar->goal = MonsterGoal::Talking;
+				break;
+			case QS_ZHAR_ANGRY:
+				zhar->talkMsg = TEXT_ZHAR2;
+				zhar->goal = MonsterGoal::Inquiring;
+				break;
+			case QS_ZHAR_ATTACKING:
+				zhar->talkMsg = TEXT_NONE;
+				zhar->goal = MonsterGoal::Normal;
+				zhar->activeForTicks = UINT8_MAX;

```

Comment:
Use std::numeric_limits?

Reply:
I simply copied it from the [ZharAI](https://github.com/diasurgical/devilutionX/blob/31b0f2147a3939feefa8eec0f97e47e1c3ca6081/Source/monster.cpp#L2717), cause it does reproduce (sync) the same state.
For consistency I wouldn't change it now and we should do a cleanup-pr that replaces all occurrences of `UINT8_MAX` with `std::numeric_limits`. What do you think? 🙂